segmentation/utils/spatial_analysis_utils.py

```diff
@@ -41,41 +41,41 @@ def calc_dist_matrix(label_map, ret=True, path=None):
         np.savez(path + "dist_matrices.npz", **dist_matrices)
 
 
-def get_pos_cell_labels(analysis_type, pheno=None, fov_data=None,
-                        thresh=None, fov_channel_data=None, cell_labels=None, col=None):
+def get_pos_cell_labels(analysis_type, pheno=None, current_fov_data=None,
+                        thresh=None, current_fov_channel_data=None, cell_labels=None, specific_marker=None):
     """Based on the type of the analysis, finds positive labels that the current match phenotype or identifies cells
     with expression values for the current maker greater than the marker threshold.
 
     Args:
-        analysis_type: type of analysis, either Cluster or Channel
-        pheno: the current cell phenotype
-        fov_data: data for the current patient
-        thresh: current threshold for marker
-        fov_channel_data: expression data for column markers for current patient
-        cell_labels: the column of cell labels for current patient
-        col: the current marker
+        analysis_type (string): type of analysis, either "Cluster" or "Channel"
```

Comment:
We should change these to be lowercase, and throw an error if cluster or channel is not the argument provided for analysis type